src/test/java/io/sealights/plugins/sealightsjenkins/buildsteps/cli/executors/PrConfigTest.java

```diff
@@ -0,0 +1,104 @@
+package io.sealights.plugins.sealightsjenkins.buildsteps.cli.executors;
+
+import hudson.model.Saveable;
+import hudson.util.DescribableList;
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.configurationtechnologies.JavaOptions;
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.configurationtechnologies.TechnologyOptions;
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.configurationtechnologies.TechnologyOptionsDescriptor;
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.entities.BaseCommandArguments;
+import io.sealights.plugins.sealightsjenkins.buildsteps.cli.entities.PrConfigCommandArguments;
+import org.junit.Assert;
+import org.junit.Test;
+
+import java.io.IOException;
+import java.util.Arrays;
+import java.util.List;
+
+import static org.mockito.Mockito.mock;
+
+public class PrConfigTest extends ConfigTest {
+    private String latestCommit = "latestCommit";
+    private String pullRequestNumber = "1";

```

Comment:
Please add "final" before each string and write them in UPPER_CASE as they are constants.

Reply:
Done